playbooks/adhoc/uninstall.yml

```diff
@@ -55,6 +55,10 @@
     - pcsd
     failed_when: false
 
+  - shell: "atomic uninstall atomic-openshift-master"
+    changed_when: False
```

Comment:
@abutcher thanks for the review.  Still it is not clear to me why this should be different than what we do with the docker containers.  We just delete them.  In fact, uninstall.yml doesn't even check if the containers exist before trying to remove them.  Who would use this information?

In my local development version I made some changes for it to work with the "origin" images as well.  This is what I have now (same thing done for -node):

```
  - shell: atomic uninstall "{{ item }}"-master
    changed_when: False
    failed_when: False
    with_items:
    - openshift-enterprise
    - atomic-enterprise
    - origin
```